src/P3Scheme.jl

```diff
@@ -219,24 +219,79 @@ end
 Γ(a::FT) where {FT <: Real} = FT(SF.gamma(a))
 
 """
-    DSD_μ(p3, λ)
+    μ_to_λ(μ)
+
+ - μ - parameter for gamma distribution of N′
+
+Returns corresponding λ to given μ value
+"""
+function μ_to_λ(p3::PSP3, μ::FT) where {FT}
+    return ((μ + p3.c) / p3.a)^(1 / p3.b)
+end
+
+"""
+    μ_approx(p3, q, N, ρ_r, F_r)
+
+ - p3 - a struct with P3 Scheme parameters
+ - q - mass mixing ratio
+ - N - ice number concentration [1/m3]
+ - ρ_r - rime density (q_rim/B_rim) [kg/m^3]
+ - F_r - rime mass fraction (q_rim/q_i)
+
+Returns linearly approximated μ between 0 and 6 
+"""
+function μ_approx(p3::PSP3, q::FT, N::FT, ρ_r::FT, F_r::FT) where {FT}
+    # Get thresholds for given F_r, ρ_r
+    th = thresholds(p3, ρ_r, F_r)
+
+    # Get min and max lambda values
+    λ_0 = μ_to_λ(p3, FT(0))
+    λ_6 = μ_to_λ(p3, p3.μ_max)
+
+    # Get corresponding q/N values at given F_r
+    q_min = log(q_gamma(p3, F_r, log(λ_0), FT(0), th))
+    q_max = log(q_gamma(p3, F_r, log(λ_6), p3.μ_max, th))
+
+    # Return approximation between them
+    return (p3.μ_max / (q_max - q_min)) * (log(q / N) - q_min)
+end
+
+"""
+    DSD_μ(p3, q, N, ρ_r, F_r)
```

Comment:
Do we need both `μ_approx` and `DSD_μ(p3, q, N, ρ_r, F_r)` ? Would it be enough to combine them and have something like `DSD_μ_approx`?